lib/Cache/Core/CoreHandler.php

```diff
@@ -270,8 +276,29 @@ public function load($key)
 
         if ($item->isHit()) {
             $data = $item->get();
+
             $data = unserialize($data);
 
+            // TODO if this change is really for release 6.8 we could simplify this and
+            // use the service instead.
+            // see  "Add service for DeepCopy/ElementDescriptor tasks", https://github.com/pimcore/pimcore/pull/6983
+            $copier = new DeepCopy();
```

Comment:
I guess we should use the new service `\Pimcore\Model\Element\Service::getDeepCopyInstance()` here

Reply:
This is not going to be part of a bug fix release anyway 😉